lib/refile/backend/file_system.rb

```diff
@@ -34,7 +34,7 @@ def initialize(directory, max_size: nil, hasher: Refile::RandomHasher.new)
       # @return [Refile::File]      The uploaded file
       verify_uploadable def upload(uploadable)
         id = @hasher.hash(uploadable)
-        IO.copy_stream(uploadable, path(id))
+        ::File.open(path(id), "wb") { |dest| IO.copy_stream(uploadable, dest) }
```

Comment:
I see. Then it depends on jruby fixes for this bug.
I'll use `Encoding.default_internal = nil` reset in my rails app as a temporarily solution for now.